generators/app/templates/src/app/screens/OnBoarding/components/Swiper/components/SecondScreen/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { View } from 'react-native';
+import i18next from 'i18next';
+import CustomText from '@components/CustomText';
+
+import styles from './styles';
```

Comment:
i think you have to import the translation file when you use `i18n.t`. Check this out in all other templates.
The import should be only of the i18n resource file corresponding to the ONBOARDING

Reply:
I imported that file in the OnBoarding index root, with importing the file in the root it's enough.